src/cloudShell.ts

```diff
@@ -178,7 +178,7 @@ export class AzureCloudShell extends BaseShell {
         const response: MessageItem = await vscode.window.showWarningMessage(message, DialogOption.ok, DialogOption.cancel);
         if (response === DialogOption.ok) {
             const accountAPI: AzureAccount = vscode.extensions
-                .getExtension<AzureAccount>("ms-vscode.azure-account")!.exports;
```

Comment:
The `ms-azuretools.vscode-azureresourcegroups` extension doesn't expose the APIs, so it will fail slightly. You can revert this change, and add a TODO or Note before showing the dialog. And this dialog should not be showed.